src/features/user/CompleteSignup/CompleteSignup.module.scss

```diff
@@ -2,7 +2,7 @@
 
 // entire page
 .signUpPage {
-  min-height: 100vh;
+  min-height: 150vh;
```

Comment:
@Shreyaschorge I don't think this solution is advisable. Revert this change and let's handle it later.

Reply:
I've added this in https://github.com/Plant-for-the-Planet-org/planet-webapp/issues/1557

Reply:
@mohitb35 The way the form is structured its the only way. Or else we need to change the entire CSS of that form.

Reply:
Yes, agreed. That's why it's better to leave that point open.